nebula_ros/schema/sub/hardware.json

```
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Hardware parameters.",
+  "type": "object",
+  "definitions": {
+    "launch_hw": {
+      "type": "boolean",
+      "readOnly": false,
+      "description": "Launch sensor hardware."
+    },
+    "rotation_speed": {
+      "type": "integer",
+      "description": "Motor RPM, the sensor's internal spin rate."
+    },
+    "sensor_model": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Sensor model."
+    },
+    "setup_sensor": {
+      "type": "boolean",
+      "default": true,
+      "readOnly": true,
+      "description": "Enable sensor setup on hardware driver."
+    },
+    "retry_hw": {
+      "type": "boolean",
+      "default": true,
+      "description": "hw driver startup retry (false when using pcap)."
```

Not inherently related to PCAPs, a more generic description like the one below would be preferable.
```suggestion
      "description": "Whether TCP connections are retried on failure or the driver should instead exit."
```